### DISCUSSION TITLE
摘要：〈關於數位人文學研究的可能性〉

### DISCUSSION THREAD

**Discussion by Taoyi-Chiang** [1101-GS4719-C/Python-Community-GS4719-C#2125] [Ideas]
數位人文從一個單純的概念逐漸演變為一門專門的學科，這樣的轉變不僅是學術社群的努力，也顯見資訊科技在人文學科的應用上具有其優勢。近年臺灣數位人文學會致力於推動數位人文學的發展，學會舉辦的諸多講座、研討會等學術活動，成為臺灣人文學者與資訊科學學者重要的交流場域。而日本近年同樣在此風潮之下，積極投入資訊科學與人文學的跨域新學科。

日本的數位文學研究歷史可追溯至1930年，以波多野完治《文章心理學》為代表。該著作首次藉由文本的統計分析，其中包含文句長度、詞性使用率等語言運用的特質，分類、比較了不同作家的風格。然而在統計技術乃至於硬體技術上的侷限，其研究成果僅能作為一種初步的嘗試。而後安本美典、村上征勝等學者則開始利用因素分析及其他資訊科學技術，進行較具深度的文體分析。

這樣的發展使得以量化分析為基礎的文學研究，多將焦點放在作者推定。這種分析方法計算特定文本中詞彙的出現頻率和句子的結構特徵，從而判斷特定文本的作者身份。這樣的研究成果足以對許多文獻真偽的問題，提出具有一定程度說服力的結論。在研究對象方面，谷崎潤一郎和志賀直哉是數位文學分析的重要範例。谷崎潤一郎的作品，特別是《春琴抄》，被用來探討文學作品中的感官描寫與文體變化。通過計量文學技術，研究者可以分析他如何運用細膩的描寫來創造其獨特的美學風格。志賀直哉的《暗夜行路》等作品則常作為簡約風格的典範，研究者透過統計分析揭示出他的作品在句子結構和語言簡潔性上的特徵。

不僅是單一作者的研究問題，學術界也開始探索新的研究視角，其中以Franco Moretti所提出的宏觀分析（macroanalysi），為文學的量化研究帶來新的契機。Moretti主張統計從宏觀層面分析文學作品的社會背景、歷史演變等狀態，可以揭示出傳統精讀（close reading）所難以察覺的文學週期與模式。例如他以18至19世紀英國小說的發展，發現文學類行的流行趨勢存在25到30年的週期性變化。而這種遠讀（distant reading）方式使我們更全面理解文學史的演進，而不僅是聚焦於少數的經典作品。中村明對於日本語文體論的問題，提到了客觀性統計的優勢在於去神秘化及提供科學化的依據。但同時中村明也指出過度依賴量化分析，也可能使研究者對文學作品的理解過於簡化。

除此之外，統計方法也能夠突顯出文體不僅是作家個人的風格問題，同時也受到時代、國家、文學類型等多重因素的影響、塑造。Matthew L. Jockers的研究持續深化遠讀的概念。藉由分析19世紀的英語小說，Jockers討論文學文本如何受到外部因素的影響。諸如此類的研究，展現了資訊科學技術在文學研究中不可限量的應用潛力。

數位人文學基於量化的思維方式，或許也得益於Roland Barthe提出的「作者之死」理論。該理論弱化了作者在文本中的主導地位，強調讀者對文本意義的重構，這一點與遠讀和宏觀分析的數位文本研究有著高度契合。此外Barthe在《神話學》中的符號學理論也常被用來解讀文學作品中的符號和隱喻，而這些符號可通過數位技術進行定量分析，以揭示文本中的深層含義。即便如此，數位人文學在文學研究的應用上仍然面臨不少的挑戰，例如如何以統計結果和文學詮釋有機的結合，又如何開發更適合文學研究的分析方法，仍然是學術界需要思考的問題。然而隨著資訊科技的入門門檻逐日降低，以及軟硬體的長足提升，均使人文學在學術產出上有相當良好的前景。

[Uploading 文学研究におけるデジタル・ヒューマニティーズの可能性.pdf…]()

**Comment by Chattenoire** [comment]
台大歷史系的陳弱水教授也有發表一些關於公共人文學的看法，個人覺得可以跟這篇相應和，不妨也看看
